dqc_us_rules/__init__.py

```diff
@@ -10,6 +10,14 @@ def run_checks(val, *args, **kwargs):
     """
     The function to run all of the validation checks under the SEC package
     """
+    if val.disclosureSystem.validationType != "EFM":
```

Comment:
Shouldn't this be something like 
```python
if 'EFM' in val.disclosureSystem.validationType
```

To handle things like efm-strict? Or is that handled by the disclosure system behavior already?